Source/utils/paths.cpp

```diff
@@ -103,6 +105,8 @@ const std::string &ConfigPath()
 		configPath = FromSDL(IOSGetPrefPath());
 #elif defined(NXDK)
 		configPath = NxdkGetPrefPath();
+#elif defined(__amigaos__)
+		prefPath = std::string("PROGDIR:");

```

Comment:
`configPath`?

Reply:
This one should also be changed to wrap `FileExistsAndIsWriteable`

Reply:
This is the `ConfigPath()` method, probably not?

Reply:
```cpp
#if !defined(__amigaos__)
		if (FileExistsAndIsWriteable("diablo.ini")) {
			prefPath = std::string("." DIRECTORY_SEPARATOR_STR);
		}
#endif
```